pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -45,6 +46,64 @@ load_user_files <- function(input, pipeline_config, prev_out, input_dir = INPUT_
   return(res)
 }
 
+#' Read Parse files
+#'
+#' @param config
+#' @param input_dir
+#'
+#' @return
+#' @export
+#'
+#' @examples
+read_parse_files <- function(config, input_dir) {
+  counts_list <- list()
+  annot_list <- list()
+  feature_types_list <- list()
+
+  samples <- config$samples
```

Comment:
Does parse data come de-multiplexed? one sample per folder? or are we offloading that work to users? What does the seurat function do about this?

Reply:
eventually we might need to use the sample column in the `cell_metadata.csv` file. but we will need to think more about this.

Reply:
Every parse experiment I've seen so far comes separated in folders. At some point in the future we could implement support for the sample column in cell_metadata but I think we need to learn more about the format beforehand.